Angolo_Attacco_Effettivo.m

```diff
@@ -0,0 +1,90 @@
+%% \file Angolo_Attacco_Effettivo
+%  \brief Calculation and visualization of the blade section angle of
+%  attack during a complete rotation of the rotor blade. 
+% \author Serena Manganiello, Michele Massa. 
+% \version 1.0
+%
+% Eli-TAARG is free software; you can redistribute it and/or
+% modify it under the terms of the GNU General Public
+% License as published by the Free Software Foundation; either
+% version 3 of the License, or (at your option) any later version.
+%
+% Eli-TAARG is developed by the TAARG Educational organization for
+% educational purposes only.
+% Theoretical and Applied Aerodynamic Research Group - University of Naples Federico II.
+%
+% Eli-TAARG GitHub link: <https://github.com/TAARG-Education/Eli-TAARG>
+%
+% Eli-TAARG is distributed in the hope that it will be useful,
+% but WITHOUT ANY WARRANTY; without even the implied warranty of
+% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+% General Public License for more details.
+% <http://www.gnu.org/licenses/>.
+%
+% ==============================================================================================
+% |Name        : Angolo_Attacco_Effettivo
+% |Author      : Serena Manganiello, Michele Massa.
+% |              University of Naples Federico II.
+% |Version     : 1.0                                                                      
+% |Date        : 01/12/2020                                                            
+% |Modified    : 16/01/2021                                                        
+% |Description : Calculation and visualization of the effective angle of attack
+%                during rotation of the rotor blade.                                             
+% |Reference   : R. Tognaccini. "Lezioni di aerodinamica dell’ala rotante" 2019 pp. 109-112.
+% |Input       : < lambda > rotor inflow ratio.
+%                < r_segn > adimensional coordinate along the rotor blade.
+%                < beta > blade flap angle.
+%                < dbeta > blade flap angle's derivative.
+%                < mu > rotor advance ratio.
+%                < psi > azimuth angle of the blade.
+%                < theta > blade pitch.
+%                < alpha_stall_up > maximum blade section angle of attack.
+%                < alpha_stall_lo > minimum blade section angle of attack.
+% |Output      : blade section angle of attack.
+% |Note        : u_R was calculated to be thorough; stall angles are
+%                imposed in order to visualize only the non-stalled region
+%                of the rotor.
+% ==============================================================================================
+
+function [alpha_e] = Angolo_Attacco_Effettivo(lambda,r_segn,beta,dbeta,mu,psi,theta,alpha_stall_up,alpha_stall_lo)

```

Comment:
English name needed.

Reply:
We added the test case and the documentation. We also changed the name of the function.